### ISSUE TITLE
Restic commands output is base64 encoded

### ISSUE THREAD

**Issue by jedimonkey** [camptocamp/bivac#327]
Had to update to the latest 2.2.0 just yesterday as something has stopped working with the old version that I was running.

The bivac commands all worked as expected (show volumes, etc). However running the restic commands to show me what snapshots were available, etc all yielded this:

```bash
bivac restic --volume dockprom_caddy snapshots
SUQgICAgICAgIFRpbWUgICAgICAgICAgICAgICAgIEhvc3QgICAgICAgICAgICAgIFRhZ3MgICAgICAgIFBhdGhzCi0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLQpmM2RlN2NiNyAgMjAxOS0xMS0xMiAxMToxOToyMCAgaXAtMTcyLTMxLTQyLTE5OSAgICAgICAgICAgICAgL3Zhci9saWIvZG9ja2VyL3ZvbHVtZXMvZG9ja3Byb21fY2FkZHkvX2RhdGEKLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tCjEgc25hcHNob3RzCg==
```

which I suspected it was base64 encoded, not some weird encryption issue so I tried dumping the output into a base 64 decoder and it worked as expected!

```bash
bivac restic --volume dockprom_caddy snapshots | base64 --decode
ID        Time                 Host              Tags        Paths
---------------------------------------------------------------------------------------------------------
f3de7cb7  2019-11-12 11:19:20  ip-172-31-42-199              /var/lib/docker/volumes/dockprom_caddy/_data
---------------------------------------------------------------------------------------------------------
1 snapshots
```

Trying other commands with restic all appear to be base64 encoded.

So there must be missing some sort of base 64 decode within bivac currently.

**Comment by cryptobioz** [comment]
We had to implement this to prevent a logs parsing issue. Which container orchestrator do you use?

**Comment by jedimonkey** [comment]
This is just a vanilla docker environment, using docker-compose to configure most containers.
I get why you may need to encode, just seems odd default behaviour (especially when other bivac commands aren't encoded). Or to not have a flag to automatically decode the output.

**Comment by cryptobioz** [comment]
The encoding should be transparent that's why there is no flag to enable or disable it. I fixed this issue with 29c335f66514ccf668ef94341e92e4f130e43adc, you can use the image `camptocamp/bivac:latest`.

**Comment by tomaaron** [comment]
@cryptobioz Maybe you can tag the image different that `latest`? I would suggest `2.2.1`. Thanks in advance.
